docs/overview.md

```diff
@@ -0,0 +1,64 @@
+# Interledger Overview
+
+Interledger enables the seamless exchange of value across different payment networks. Traditional payment networks 
+operate independently from each other. Sending value is easy only if the sender and recipient have accounts on the same 
+network, but it can be slow and expensive if they have accounts on different networks.
+
+Interledger is not tied to any one company, blockchain, or currency. You can easily send XRP to someone who wants to 
+receive ETH, or you can send USD through a Paypal account to someone who wants to receive USD in a Venmo account. 
+Interledger makes it easy to transact in whatever currency or payment network you choose.
+
+## What is Interledger?
+Interledger is a network of computers that enables sending value across independent payment networks. Similar to the Internet, 
+Interledger routes packets of value from source to destination. Computers on the Interledger network are 
+called *nodes*. Nodes can take one or more of the following roles: 
+* Sender Node – Initiates a value transfer
+* Router Node – Applies currency exchange and forwards packets of value. This is the intermediary node between 
+the sender and the receiver. 
+* Receiver Node – Receives and completes the value transfer
```

Comment:
The receiver doesn't actually "complete" the transfer. It accepts it but it is only complete end-to-end when the fulfilment gets back to the sender.

Reply:
https://github.com/interledger/interledger.github.io/pull/113/files#diff-f3b56000093113bd3bfb6c9c05e7e945R18